libs/application/templates/inheritance-report/src/lib/utils/helpers.ts

```diff
@@ -8,3 +10,13 @@ export const currencyStringToNumber = (str: string) => {
 
 export const isValidString = (string: string | undefined) =>
   string && /\S/.test(string)
+
+const emailRegex =
+  /^[\w!#$%&'*+/=?`{|}~^-]+(?:\.[\w!#$%&'*+/=?`{|}~^-]+)*@(?:[A-Z0-9-]+\.)+[A-Z]{2,6}$/i
+
+export const isValidEmail = (value: string) => emailRegex.test(value)
```

Comment:
Should we write tests for these?
Just a spec file with a couple of tests for sanity?

Reply:
Simple as: https://github.com/island-is/island.is/blob/main/libs/application/templates/estate/src/lib/utils.spec.ts